cluster-autoscaler/simulator/cluster_snapshot.go

```diff
@@ -39,6 +39,8 @@ type ClusterSnapshot interface {
 	RemovePod(namespace string, podName string, nodeName string) error
 	// AddNodeWithPods adds a node and set of pods to be scheduled to this node to the snapshot.
 	AddNodeWithPods(node *apiv1.Node, pods []*apiv1.Pod) error
+	// IsPVCUsedByPods returns if the pvc is used by any pod
```

Comment:
nit: maybe worth noting here that `key` is supposed to be in `<namespace>/<pvc_name>` format.

Reply:
done